### ISSUE TITLE
Possible bug in tutorial code-samples

### ISSUE THREAD

**Issue by saurabhnanda** [tathougies/beam#257]
Am I missing something very obvious or are almost all code-samples in the [Aggregates chapter](https://tathougies.github.io/beam/user-guide/queries/aggregates/) missing join-conditions. For example:

```
SELECT "t0"."GenreId" AS "res0",
       "t0"."Name" AS "res1",
       COUNT("t1"."TrackId") AS "res2"
FROM "Genre" AS "t0"
INNER JOIN "Track" AS "t1"
GROUP BY "t0"."GenreId",
         "t0"."Name";
```

Shouldn't there be some sort of PK-FK relationship associating genre with a track?

**Comment by tathougies** [comment]
Many of them are, but this is not a bug. The beam code samples themselves were not written to take advantage of the join condition. I'm not sure why I didn't add them, but I'll go through and rejigger them to make them work correctly. It may have been the result of find/replace or an automated code change.

**Comment by tathougies** [comment]
Fixed in the latest `beam-0.8.0.0` branch
